### DISCUSSION TITLE
http/Encoding

### DISCUSSION THREAD

**Discussion by giscus** [m6w6/mdref-http#8] [Comments]
# http/Encoding

http://localhost:8000/http/Encoding

**Comment by kimboslice99** [comment]
Info here seems to be quite out of date? propro is no longer maintained

Futhermore, the web is littered with people unable to get it to work

Nobody has definitive information on the installation and use of this package, it would seem.

**Reply by m6w6** [reply]
Is giscus broken, or did you really post this to `http\Encooding`?

> propro is no longer maintained

ext-propro _is_ maintained. Zend Engine v4 (i.e. PHP 8) dropped internal API (get and set object handlers) this extension was utilising.

From https://mdref.m6w6.name/http :

> This extension unconditionally depends on the pre-loaded presence of the following PHP extensions:
>  * raphf
>  * propro (only until < 4.0.0, resp. PHP 8)
>  * spl
